OmfIngressClientLibraries/Program.cs

```diff
@@ -103,7 +109,23 @@ public static void Setup()
         public static async Task<OmfConnection> CreateOmfConnectionAsync()
         {
             // Create the Connection
-            OmfConnection omfConnection = await _omfIngressClient.CreateOmfConnectionAsync(DeviceClientId, ConnectionName, NamespaceId).ConfigureAwait(false);
+            // Begin creation of OmfConnection
+            Console.WriteLine($"Creating an OMF Connection in Namespace {NamespaceId} for Client with Id {DeviceClientId}");
+            Console.WriteLine();
+            OmfConnectionCreate omfConnectionCreate = new ()
+            {
+                Name = ConnectionName,
+                Description = "This is a sample Connection",
+            };
+            omfConnectionCreate.ClientIds.Add(DeviceClientId);
+            OmfConnection omfConnection = await _omfIngressService.BeginCreateOmfConnectionAsync(omfConnectionCreate).ConfigureAwait(false);
+
+            while (!omfConnection.State.Equals("Active"))
```

Comment:
maybe comment to explain what we are doing, and maybe a failsafe in case it never goes Active